### PR TITLE
[ENH] Document how to give feedback on documentation and add issue template

### LINKED ISSUE #125: Put explanation for how to give feedback on docs
https://neurobagel.org/getting_help/

essentially: make giving feedback as obvious and easy as practically possible

TODO:
- [x] link to documentation repo issues page
- [x] add a custom issue template for this repo for documentation issues that includes
  - [x] link to the page in the docs that is missing info or inaccurate (if applicable)
  - [x] detail(s) that could be clarified

### PR DESCRIPTION
**Pull request by alyssadai**
Closes #125 

## Checklist

- [x] PR has an interpretable title with a prefix (`[ENH]`, `[FIX]`, `[REF]`, `[TST]`, `[CI]`, `[MNT]`, `[INF]`) _(see https://neurobagel.org/contributing/pull_requests for more info)_
- [x] PR links to GitHub issue with mention `Closes #XXXX`
- [x] Checks pass

### CODE REVIEW

File: docs/getting_help.md [hunk 1]
```diff
@@ -1,7 +1,13 @@
 # Getting Help
 
+## Report a bug or request a feature
 For a bug or feature request, feel free to open an issue in the repository for the relevant tool in the [Neurobagel GitHub organization](https://github.com/neurobagel).
 
-If you are using one of our web tools ([https://query.neurobagel.org](https://query.neurobagel.org) or [https://query.neurobagel.org](https://annotate.neurobagel.org)), you can also submit suggestions and bugs using the feedback widget directly on the site.
+If you are using one of our web tools ([https://annotate.neurobagel.org](https://annotate.neurobagel.org) or [https://query.neurobagel.org](https://query.neurobagel.org), you can also submit suggestions and bugs using the feedback widget directly on the site.
 
+## Documentation feedback
+Neurobagel is always looking to improve their user documentation.
```
Comment (surchs) [inline docs/getting_help.md:9]:
```suggestion
We are always looking to improve our user documentation.
```
This is us talking about ourselves, no?

File: docs/getting_help.md [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
+To request a change or addition to existing documentation, please [open an issue in the `neurobagel/documentation` repo](https://github.com/neurobagel/documentation/issues).
```
Comment (surchs) [inline docs/getting_help.md:10]: I'm guessing there is no easy way to link directly to the nice new issue template you made?

**Review by surchs — APPROVED** [review]
Very nice!

One :cherries: comment,
then :cook:

File: docs/getting_help.md (hunk 2 above)
Comment (alyssadai) [reply docs/getting_help.md:10]: I could link to this page https://github.com/neurobagel/documentation/issues/new/choose which should update to show the new issue template once the PR is merged.

However, I think I would still prefer to direct the user to the list of current issues here so they can first see which topics/changes have already been requested, etc.

Let me know if you feel differently.

Comment (surchs) [reply docs/getting_help.md:10]: good point, agreed

**Pull request merged**